apps/aehyperchains/test/aehc_consensus_tests.erl

```diff
@@ -626,8 +627,8 @@ make_spend_tx(Sender, SenderNonce, Recipient, Amount) ->
                                        payload => <<>>}),
     SpendTx.
 
-staking_contract_vm() -> 5.
-staking_contract_abi() -> 3.
+staking_contract_vm() -> ?VM_FATE_SOPHIA_2.
+staking_contract_abi() -> ?ABI_FATE_SOPHIA_1.
```

Comment:
This is exactly what I'm working on right now. We should coordinate efforts more.